provisioner_test.go

```diff
@@ -119,3 +119,44 @@ func TestNewDisembedderProvisioner(t *testing.T) {
 	assert.NoError(t, err)
 	testProvisionerSuccessful(t, *p, "linux", "amd64", DefaultVersionAstilectron, DefaultVersionElectron)
 }
+
+func TestZipShouldRemove(t *testing.T) {
+	var o = Options{
+		DataDirectoryPath: mockedTempPath(),
+	}
+
+	// Make sure the test directory doesn't exist.
+	if err := os.RemoveAll(o.DataDirectoryPath); err != nil && !os.IsNotExist(err) {
+		t.FailNow()
+	}
+	defer os.RemoveAll(o.DataDirectoryPath)
+
+	a, err := New(astikit.AdaptTestLogger(t), o)
+	if err != nil {
+		t.Fatalf("main: creating astilectron failed: %s", err.Error())
+	}
+
+	p := a.Paths()
+
+	if err = a.provision(); err != nil {
+		t.FailNow()
```

Comment:
Could you replace this with `t.Fatalf("main: provisionning failed: %s", err.Error())`

Reply:
OK. https://github.com/asticode/go-astilectron/pull/358/commits/6daec70338071bbd144191b10d6b33d200e1a142#diff-f5c6513a93e0d3a0d10fb30654a66c3ab4b7f05ed5b4dd9dd10ef379a14496adR142